### PR TITLE
Combine map filter Desktop

### PR DESCRIPTION
**Pull request by jojudre**
PR Type
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Description

Combine map filter Desktop

## Git Issues 
 
 related issue - [#1411](https://github.com/ONEARMY/community-platform/issues/1411)
 
 

https://user-images.githubusercontent.com/42969746/151658714-8e9e6962-609f-4c1e-b059-c66aecf1d34f.mp4

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -58,7 +58,6 @@
     "terser-webpack-plugin": "4.2.3"
   },
   "dependencies": {
-    "@khanacademy/react-multi-select": "^0.3.3",
```
Comment (thisislawatts) [inline package.json:61]: **praise:** One less dependency 🥳 Great to get this deprecated component removed.

**Review by thisislawatts — CHANGES_REQUESTED** [review]
**nitpick(blocking):** Overall this looks great, thank you! One detail you will need to adjust is the commit message so that it aligns with the formatting convention we use on this project.

Consider something  like:
`feat: combine desktop workshop filters to match mobile filters`

**Comment by davehakkens** [comment]
Looks nice! Like seeing the multiple filters in there. Thanks a lot!

**Review by thisislawatts — APPROVED** [review]

**Comment by thisislawatts** [comment]
Thanks so much @jojudre for taking the time to raise this PR. I pushed a commit addressing the `yarn.lock` conflicts and added your changes via [this PR](https://github.com/ONEARMY/community-platform/pull/1423). I have had some time today to merge some long standing PRs so ended up causing a lot of churn in our `yarn.lock` file.

I will mark this PR as closed as [your commit is now on the way out to production](https://app.circleci.com/pipelines/github/ONEARMY/community-platform/1257/workflows/1d06fbf9-b9fd-4b22-96c4-a2c4a27beded).

Let me know if you've got any questions!

**Pull request closed without merging**